src/platform/Kconfig

```diff
@@ -567,6 +567,13 @@ config HAVE_AGENT
 	  with DMA based scheduling, where asynchronous interrupts
 	  can potentially starve the agent.
 
+config AMS
+	bool "Enable Async Messaging Service"
+	default n
+	help
+	  Enables Async Messaging Service.
+	  Async messages are used on ACE to send messages between modules.
```

Comment:
is there any hardware dependency? If not, then it's fine to select this for ACE but the module itself should not be tagged as ACE only.

if there is a dependency, then use "depends on ACE"

Reply:
The target platform for this functionality was ACE, but it's purely software implementation. Removed the ACE part to avoid confusion.